www/include/RunResultHtmlTable.php

```diff
@@ -52,7 +52,8 @@ public function __construct($testInfo, $runResults, $rvRunResults=null) {
     $this->enabledColumns[self::COL_ABOVE_THE_FOLD] = $testInfo->hasAboveTheFoldTime();
     $this->enabledColumns[self::COL_RESULT] = true;
     $this->enabledColumns[self::COL_CERTIFICATE_BYTES] = $runResults->hasValidNonZeroMetric('certificate_bytes');
-    $checkByMetric = array(self::COL_USER_TIME, self::COL_DOM_TIME, self::COL_TTI, self::COL_SPEED_INDEX,
+    $this->enabledColumns[self::COL_FIRST_CONTENTFUL_PAINT] = true;
```

Comment:
FCP is only available for Chrome. Can you change this to only enable if the metric is present in the result (like certificate bytes above it)?